src/pages/ConvertVrt/Form/index.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Formik, FormikProps, FormikConfig } from 'formik';
+
+import getValidationSchema, { FormValues } from './validationSchema';
+
+export * from './validationSchema';
+
+export interface IAmountFormProps
+  extends Omit<FormikConfig<FormValues>, 'onSubmit' | 'initialValues'> {
+  onSubmit: (values: { vrt: string; xvs: string }) => Promise<void> | void;
+  children: (formProps: FormikProps<FormValues>) => React.ReactNode;
+  initialAmount?: string;
+  maxVrt: FormikConfig<FormValues>['initialValues']['vrt'];
+  maxXvs: FormikConfig<FormValues>['initialValues']['xvs'];
+}
+
+export const ConvertVrtForm: React.FC<IAmountFormProps> = ({
+  children,
+  onSubmit,
+  initialAmount = '',
+  maxVrt,
+  maxXvs,
+}) => {
+  const handleSubmit = (values: FormValues) => {
+    onSubmit(values);
+  };
+
+  return (
+    <Formik
+      initialValues={{
+        vrt: initialAmount,
+        xvs: initialAmount,
+      }}
+      onSubmit={handleSubmit}
+      validationSchema={getValidationSchema({ maxXvs, maxVrt })}
+      validateOnMount
+      validateOnChange
+    >
+      {children}
```

Comment:
I'd suggest passing the formik props tot the `children`, you'll likely need them when you do the wire up work

Reply:
We don't need them anymore if we use the formik hooks like in the previous PR

Reply:
In some instances you'll need access to the Formik props without using a hook to read the context, for example to create a function that uses the values, like we do with `calculateDailyEarnings` on the Borrow/Repay form.